### PR TITLE
Docs revamp

### PR DESCRIPTION
**Pull request by discombobulateme**
Working in progress to revamp Docs page based on new design: https://www.figma.com/file/gxXWfTFbeSAESpVU0pGsly/Doc-style-changes?node-id=60%3A608

Several changes are being done including:
- [ ]  including variables files to create standard styles
- [ ] adding partial files to better organize the styles
- [ ] creating new classes to match design

### CODE REVIEW

**Review by kimsauce — CHANGES_REQUESTED** [review]
Not sure if it's just me -- I couldn't get the build to work :(
Screen Shot 2022-04-06 at 3 57 00 PM

**Comment by discombobulateme** [comment]
@kimsauce did you `npm install` ? packages need to be reinstall as it is using the breadcrumbs

File: src/css/admonitions.css [hunk 1]
```diff
@@ -2,21 +2,31 @@
 /* Callout boxes / admonition styles */
 
 .alert a {
-    color: #E2231A;
+    color: var(--ds-red-600);
```
Comment (spider-sauce) [inline src/css/admonitions.css:5]: good use of variables 👍

File: src/css/fonts/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
+@font-face {
+    font-family: 'Proxima Nova Bl';
+    src: url('ProximaNova-Black.eot');
+    src: local('Proxima Nova Black'), local('ProximaNova-Black'),
```
[the remark this answers is not in the document]
Comment (discombobulateme) [reply src/css/fonts/stylesheet.css:4]: Yes! I just couldn't find someone to actually give me the Sauce certificate number or... something like... so I added the font face

File: docusaurus.config.js [hunk 3]
```diff
@@ -12,35 +12,22 @@ const docusaurusConfig = {
...
-      "/scripts/hide.js",
+    '/scripts/hide.js',
+    // Need Help? button
     {
       src: 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js',
       type: 'text/javascript',
       charset: 'UTF-8',
       'data-domain-script': '9e4c4ce3-8349-4030-9de7-0b1d368edfce',
     },
     {
```
[the remark this answers is not in the document]
Comment (discombobulateme) [reply docusaurus.config.js:23]: :O thank you

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@discombobulateme @spider-sauce approved/merged!

**Pull request merged**